frontend/test/metabase/scenarios/visualizations/table.cy.spec.js

```diff
@@ -156,6 +156,8 @@ describe("scenarios > visualizations > table", () => {
       .first()
       .click();
 
+    cy.wait("@dataset");
```

Comment:
You were on the right path here.
This line alone was all that's needed to fix this flake.

But as I said, the current test is wasteful and quite taxing. I'd like to simplify it in another PR if that's ok with you @diogormendes ?